src/Utils/protected/Utils/MgxNumeric.h

```diff
@@ -70,9 +70,17 @@ namespace Math
 
 		static inline bool isNearlyEqual(const double a, const double b)
 		{
-			double s = std::fabs(a) + std::abs(b);
-			double d = a - b;
-			return (d==0.0) ? true : isNearlyZero(d/s);
+            if(a == b) {
```

Comment:
The indentation looks odd.

Reply:
Yeah, we will shortly put clang-formatting in place